classes/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] @Amazinggracee great work initializing the rental class with @date, @book, and @person, however, you were supposed to add the 'belongs-to' association between a rental and the date and book eg ```book.rentals << self```, kindly consider adding the association

Reply:
Okay! Thanks a million. updated.